File: schemaorg/tvepisode.go
package schemaorg

import "encoding/json"

// TVEpisode see : https://schema.org/TVEpisode
type TVEpisode struct {

typeContext

Episode

// CountryOfOrigin see : /countryOfOrigin
// The country of the principal offices of the production company or individual responsible for the movie or program.
CountryOfOrigin *Country `json:"countryOfOrigin"`

// SubtitleLanguage see : /subtitleLanguage
// Languages in which subtitles/captions are available, in IETF BCP 47 standard format (see: https://schema.orghttp://tools.ietf.org/html/bcp47).
SubtitleLanguage interface{} `json:"subtitleLanguage"` // types : Language Text

}

func (v *TVEpisode) MarshalJSON() ([]byte, error) {
	v.C = "http://schema.org"
	v.T = "TVEpisode"

	return json.Marshal(v)
}

File: schemaorgphp/classes/tv_episode.php
<?php

class TVEpisode extends Episode implements JsonSerializable {

	public static $context = 'http://schema.org';
	public static $type = 'TVEpisode';
	
	/**
	 * The country of the principal offices of the production company or individual responsible for the movie or program.
	 * see : https://schema.org/countryOfOrigin
	 * @var \Country|\Country[]
	 */
	public var $country_of_origin;
	
	/**
	 * Languages in which subtitles/captions are available, in IETF BCP 47 standard format (see: https://schema.orghttp://tools.ietf.org/html/bcp47).
	 * see : https://schema.org/subtitleLanguage
	 * @var \Language|\Language[]|string|string[]
	 */
	public var $subtitle_language;
	
	public function jsonSerialize() {
		$out = array(
			'@context' => 'http://schema.org',
			'@type' => 'TVEpisode'
		);
		
		$serialized = so_json_serialize( $this->country_of_origin );
		if ( ! empty( $serialized ) ) {
			$out['countryOfOrigin'] = $serialized;
		}
		
		$serialized = so_json_serialize( $this->subtitle_language );
		if ( ! empty( $serialized ) ) {
			$out['subtitleLanguage'] = $serialized;
		}
		
		return $out;
	}
}
